mesonbuild/compilers/compilers.py

```diff
@@ -73,6 +73,9 @@
 # List of languages that by default consume and output libraries following the
 # C ABI; these can generally be used interchangeably
 clib_langs = ('objcpp', 'cpp', 'objc', 'c', 'fortran',)  # type: T.Tuple[str, ...]
+# List of assembler suffixes that can be linked with C code directly by the linker
+assembler_suffixes = tuple() # type: T.Tuple[str, ...]
+assembler_suffixes += ('s', 'S')
```

Comment:
The first line is not neccessary. You can just do `assembler_suffixes: T.Tuple[str, ...] = ('s', 'S')`

Reply:
Issue resolved in latest commit.